Backend/app/dashboard/CommonAPIs/src/Dashboard/ProviderPlatform/Driver.hs

```diff
@@ -764,6 +766,17 @@ type AddVehicleForFleetAPI =
 
 ---------------------------------------------------------
 
+-- add vehicle for fleet withoutDriver ------------------------------------------
+
+type AddVehicleForFleetWithoutDriverAPI =
```

Comment:
We discussed, not to have addVehicle API for fleet owner, right? Why is it there?

Reply:
So it is not exactly add vehicle , it is like first checking all weather that rc have association with any driver or fleet , if yes throwing error , otherwise calling verifyRCFlow. Changing name to `registerRCForFleetWithoutDriver`